package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "openfin-service-async",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "check": "svc-tools check",
+    "clean": "rimraf dist",
+    "build": "npm run clean && tsc -p tsconfig.json && node src/fileCopy.js",
+    "deploy:local": "npm run build && cd dist && npm pack",
+    "fix": "svc-tools fix",
+    "test": "npm run test:unit",
+    "test:unit": "svc-tools test unit"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/HadoukenIO/service-async"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/HadoukenIO/service-async/issues"
+  },
+  "homepage": "https://github.com/HadoukenIO/service-async",
+  "devDependencies": {
+    "@types/openfin": "^43.0.1",
```

Comment:
Don't think you need this dependency right?

Reply:
Also the js-adapter isn't used anywhere that I can see.

Reply:
`build` fails without `@types/openfin`, `fix`/`check` fails without `hadouken-js-adapter`

Reply:
Hmmm. That implies that they should be a sub dependency of something, since they're not used directly.

Reply:
It's service-tooling that needs them, which has them as dev dependencies. FDC3 and Notifications also have them as dev dependencies and uses them directly.

Is suspect how things are now may be down to weirdness around a package both directly and indirectly depending on two different versions of the same package, but will come back to in the morning